### PR TITLE
Setup google analytics

### PR DESCRIPTION
**Pull request by danmitchell-**
Setup as per instructions from @dan-gilbert

### CODE REVIEW

File: app/views/layouts/_analytics.html.erb [hunk 1]
```diff
@@ -0,0 +1,17 @@
+<!-- Google Analytics -->
+<script>
+
+(function(i,s,o,g,r,a,m){i['GoogleAnalyticsObject']=r;i[r]=i[r]||function(){
+(i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),
+m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)
+})(window,document,'script','https://www.google-analytics.com/analytics.js','ga');
+
+ga('create', <%= ENV['UA_TRACKING_ID'] %>, 'auto');
```
Comment (gidsg) [inline app/views/layouts/_analytics.html.erb]: load this from Rails config rather than directly from `ENV`

File: manifest-staging.yml [hunk 2]
```diff
@@ -8,5 +8,6 @@ services:
   - managing-registers-staging
   - logit-ssl-drain
 env:
-  RAILS_ENV: staging  
-  RACK_ENV: staging  
+  RAILS_ENV: staging
+  RACK_ENV: staging
+  UA_TRACKING_ID: UA-86101042-6
```
Comment (gidsg) [inline manifest-staging.yml]: should this live in the CUPS service, rather than in Travis?

Comment (danmitchell-) [reply manifest-staging.yml]: The arent secrets so i'm fine with setting them at app level

File: app/views/layouts/_analytics.html.erb (hunk 1 above)
Comment (danmitchell-) [reply app/views/layouts/_analytics.html.erb]: Ahhhh yes. My bad

File: config/secrets.yml [hunk 3]
```diff
@@ -39,6 +39,7 @@ staging:
   notify_custodian_invite_template: <%= ENV["NOTIFY_CUSTODIAN_INVITE_TEMPLATE"] %>
   notify_advanced_invite_template: <%= ENV["NOTIFY_ADVANCED_INVITE_TEMPLATE"] %>
   notify_basic_invite_template: <%= ENV["NOTIFY_BASIC_INVITE_TEMPLATE"] %>
+  google_analytics_tracking_id: <%= ENV['UA_TRACKING_ID'] %>
```
Comment (gidsg) [inline config/secrets.yml]: as this isn't a secret can we just set it in the normal config file e.g. `staging.rb`?

Comment (danmitchell-) [reply config/secrets.yml]: Rails convention is `ENV['foo']`, secrets was introduced in Rails 4.1 to keep "things" in one place

**Pull request merged**